src/PharoLauncher-CLI/PhLCliCommand.class.st

```diff
@@ -117,6 +117,16 @@ PhLCliCommand >> execute [
 	^ self context exitSuccess
 ]
 
+{ #category : #'command execution' }
+PhLCliCommand >> executeOSShellCommand: cmdString [
+
+ ^ PhLProcessWrapper new 
+		shellCommand;
+		addArguments: cmdString;
```

Comment:
I do not like using the #arguments: method for building commands because you need to be sure there is no whitespace inside your command arguments (i.e. path with a space). I prefer to use #addArgument: many times.

Reply:
Ok, will change method to use argument array.

Reply:
Hi Christophe (@demarey)! Sorry for taking some time, I have problem to determine what is command itself and what are the arguments of following: 
`kill $(pgrep -a -f Pharo | grep -v 11263 | tr -s ' ' | cut -d ' ' -f1)> /dev/null`
in order to use as following, e.g.:
```
	process := PhLProcessWrapper new 
		command: '/home/me/Pharo/pharo';
		addArgument: '/tmp/ba r.image' surroundedByDoubleQuotes;
		prepareProcess;
		yourself.
``` 
is command in this case supposed to be 'kill' and everything within `$(...)` is one argument) and `>` `/dev/null` are additional arguments? Thanks for insinght!

Reply:
@Bajger Sorry for the late answer. I did not have in mind that the command is so complicated. It is a sequence of commands using pipes. In this case, it will be difficult to use correctly PhLProcessWrapper: pipes are not managed.
You can use the one-line command in this case. There is no argument that should be protected with quotes.

Reply:
I've put arg array there, but content within braces is passed as one big argument